ihatemoney/api.py

```diff
@@ -186,8 +186,21 @@ def delete(self, project, bill_id):
         return "OK", 200
 
 
+class TokenHandler(Resource):
+    method_decorators = [need_auth]
+
+    def get(self, project):
+        print(project)
```

Comment:
It seems that this print shouldn't be here :-)

Reply:
You are right!